plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -125,45 +132,38 @@ export class PgInstrumentation extends InstrumentationBase<PgInstrumentationConf
   }
 
   protected init() {
-    const modulePG = new InstrumentationNodeModuleDefinition(
-      'pg',
-      ['>=8.0.3 <9'],
-      (module: any) => {
-        const moduleExports: typeof pgTypes =
-          module[Symbol.toStringTag] === 'Module'
-            ? module.default // ESM
-            : module; // CommonJS
-        if (isWrapped(moduleExports.Client.prototype.query)) {
-          this._unwrap(moduleExports.Client.prototype, 'query');
-        }
+    const SUPPORTED_PG_VERSIONS = ['>=8.0.3 <9'];
 
-        if (isWrapped(moduleExports.Client.prototype.connect)) {
-          this._unwrap(moduleExports.Client.prototype, 'connect');
-        }
+    const modulePgNativeClient = new InstrumentationNodeModuleFile(
+      'pg/lib/native/client.js',
+      SUPPORTED_PG_VERSIONS,
+      this._patchPgClient.bind(this),
+      this._unpatchPgClient.bind(this)
```

Comment:
can you clarify why are you patching just to immediately unpatch? I'm not understanding the need of the unpatch part here

Reply:
I moved to `InstrumentationNodeModuleFile` from `InstrumentationNodeModuleDefinition` as we need to patch the client file instead. Just passing the patch and unpatch methods with bound context. So `InstrumentationNodeModuleFile` will handle the unpatch when needed. They are passed as arguments.

Reply:
oh I'm so sorry, I read `;` instead of `,`, and it looked like you're making those calls... 🤦 
never mind, you're correct, sorry abut that